tox.ini

```diff
@@ -35,3 +36,35 @@ deps =
     pylint
 whitelist_externals =
     sh
+
+[testenv:linters]
+skip_install = true
+basepython = python3
+deps = flake8
+       flake8-colors
+       black
+commands =
+    # do not fail for black
+    -black --skip-string-normalization --check .
+    flake8
+
+[testenv:integration]
+usedevelop = true
+deps = -rintegration_tests/test-requirements-for-tox.txt
+changedir = integration_tests
+passenv =
+    WAZO_TEST_DOCKER_OVERRIDE_EXTRA
+commands =
+    make test-setup
+    pytest {posargs}
+whitelist_externals:
+    make
+
+[flake8]
+exclude =
+    .tox
+    .eggs
+    integration_tests/assets/scripts
+show-source = true
```

Comment:
You should add at least those ignore to make flake8 compatible with black:
```
# W503: line break before binary operator
# E203: whitespace before ':' warnings
ignore = E203, W503
```

Reply:
why E203?

Reply:
black adds a space before the `:` while flake8 complains you should remove it: https://github.com/python/black/issues/315

Reply:
```python
return backends[offset : limit + offset]
```
This is an example where I've seen add a space before a `:`

Reply:
It is possible to make the inverse: tell to black to ignore E203?